architects/data/ir_ui_view.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="default_map_view_for_project_architects" model="ir.ui.view">
+        <field name="name">Default.map.view.for.project</field>
+        <field name="model">project.project</field>
+        <field name="type">map</field>
+        <field name="arch" type="xml">
+            <map res_partner="x_associated_location" />
+        </field>
+    </record>
+    <record id="crm_lead_kanban_customization_architects" model="ir.ui.view">
+        <field name="name">crm.lead.kanban.customization.architects</field>
+        <field name="model">crm.lead</field>
+        <field name="type">kanban</field>
+        <field name="inherit_id" ref="crm.crm_case_kanban_view_leads" />
+        <field name="mode">extension</field>
+        <field name="priority">320</field>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//field[@name='expected_revenue']" position="after">
+                    <field name="x_request_date" display="full" />
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="crm_lead_form_customization_architects" model="ir.ui.view">
+        <field name="name">crm.lead.form.customization.architects</field>
+        <field name="model">crm.lead</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="crm.crm_lead_view_form" />
+        <field name="mode">extension</field>
+        <field name="priority">160</field>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//form[1]/sheet[1]/group[1]" position="after">
+                    <group name="studio_group_h7X3q">
+                        <group name="studio_group_h7X3q_left" string="Request Details">
+                            <field name="x_request_date"/>
+                            <field name="x_project_expected_start_date" string="Project expected start date" />
+                            <field name="x_project_type" string="Project type" />
```

Comment:
No need for string here, as same as field description... Please apply everywhere it is needed